### PR TITLE
Added functionality to copy specific columns in applicants table.

### PR DESCRIPTION
**Pull request by jeffreyzhao5618**

### CODE REVIEW

**Review by jialuogao — COMMENTED** [review]
For delimiter, please change to Tab (\t) thus TVS instead of CVS since it works well in both excel and google sheet for copying

File: views/project/applicants.ejs [hunk 1]
```diff
@@ -18,11 +18,48 @@
...
+
+                function copyColumns(colsArray) {
+                    let $temp = $("<textarea>");
+                    let copyString = '';
+
+                    $("#projectTbody tr").each(function() {
+                        for(let i = 0; i < colsArray.length; i++) {
+                            copyString += $(`td:nth-of-type(${colsArray[i] + 1})`,this).text();
+                            if(i != colsArray.length - 1) {
+                                copyString += ","
```
Comment (jialuogao) [inline views/project/applicants.ejs:47]: Change to `\t` would result in better experience

File: views/project/applicants.ejs [hunk 2]
```diff
@@ -18,11 +18,48 @@
...
+                    })
+                    if(colsArray.length > 0) {
+                        copyColumns(colsArray);
+                    }
+                })
+
+                function copyColumns(colsArray) {
+                    let $temp = $("<textarea>");
+                    let copyString = '';
+
```
Comment (jialuogao) [inline views/project/applicants.ejs:42]: Also add another option (check box) to choose if copy the headers please, copy with multiple column really needs header

**Review by jialuogao — CHANGES_REQUESTED** [review]
Very good overall, only 2 minor things to fix

**Review by jeffreyzhao5618 — COMMENTED** [review]
Added copy headers option to copy table.
Copy table now copies in TSV format.
Changed appearance of copy table.

**Review by jialuogao — APPROVED** [review]
Good, did some minor layout changes

**Pull request closed without merging**